features/prompt_memes_b/challenge_promptmeme_posting_fills.feature

```diff
@@ -46,6 +46,39 @@ Feature: Prompt Meme Challenge
     And 1 email should be delivered to "my1@e.org"
 # TODO: when work_anonymous is implemented, test that the prompt filler can be anon too
 
+  Scenario: Translated prompt notification email is sent
+  Given a locale with translated emails
+    And basic tags
+    And the following activated user exists
+      | login   | password | email     |
+      | myname1 | password | my1@e.org |
+    And the user "myname1" enables translated emails
+    And a fandom exists with name: "GhostSoup", canonical: true
+    And I am logged in as "mod1"
+    And I set up a basic promptmeme "The Kissing Game"
+  When I am logged in as "myname1"
+    And I go to "The Kissing Game" collection's page
+    And I follow "Sign Up"
+    And I fill in "challenge_signup_requests_attributes_0_tag_set_attributes_fandom_tagnames" with "GhostSoup"
+    And I check "challenge_signup_requests_attributes_0_anonymous"
+    And I press "Submit"
+  Then I should see "Sign-up was successfully created"
+  When I am logged in as "myname2"
+    And I go to "The Kissing Game" collection's page
+    And I follow "Prompts (1)"
+    And I press "Claim"
+  Then I should see "New claim made"
+    And I follow "Fulfill"
+    And I check "No Archive Warnings Apply"
+    And I select "English" from "Choose a language"
+    And I fill in "Fandoms" with "GhostSoup"
+    And I fill in "Work Title" with "Kinky Story"
+    And I fill in "content" with "Story written for your kinks, oh mystery reader!"
+  When all emails have been delivered
+    And I press "Post"
+  Then 1 email should be delivered to "my1@e.org"
+    And the email to "myname1" should be translated
```

Comment:
Can you check that this is the right email, so subject/or some part of the content?